src/app/main/component/user/components/habit/all-habits/all-habits.component.ts

```diff
@@ -78,6 +79,18 @@ export class AllHabitsComponent implements OnInit, OnDestroy {
     this.masterSubscription.add(habitServiceSub);
   }
 
+  public checkHabitsView(): void {
+    const galleryView = this.localStorageService.getHabitsGalleryView();
+    switch (galleryView) {
+      case null:
+        this.localStorageService.setHabitsGalleryView(true);
+        break;
+      case 'false':
+        this.galleryView = false;
+        break;
+    }
```

Comment:
```suggestion
    const galleryView = this.localStorageService.getHabitsGalleryView();
    this.galleryView = galleryView ?? this.galleryView;
```
No need to save it to LS here, default value is defined anyways.